test/test_nn.py

```diff
@@ -15331,6 +15331,7 @@ def test_softmax_results(self, device, dtype):
                         self.assertEqual(input.grad, ref_input.grad)
 
     @onlyCUDA
+    @dtypes()  # since dtypes is mandatory
```

Comment:
For these cases let's replace `@dtypesIfCUDA` with `@dtypes`

Reply:
Ofcourse! Should have thought about that! Thank you very much!